pkgs/data/fonts/nerdfonts/update.sh

```diff
@@ -20,7 +20,7 @@ while
   read -r name
   read -r url
 do
-    printf '  "%s" = "%s";\n' "${name%.*}" "$(nix-prefetch-url "$url")" >>"$dirname/shas.nix"
-done < <(jq -r '.assets[] | .name, .browser_download_url' <<<"$latest_release")
+    printf '  "%s" = "%s";\n' "${name%%.*}" "$(nix-prefetch-url "$url")" >>"$dirname/shas.nix"
```

Comment:
This change is to ensure we split on the first delimiter rather than the last, without this we'll end up with font names with `.tar` suffixes.